OnLand/ControlLaptop/templates/_cards/cells1.html

```diff
@@ -7,294 +7,14 @@
                 <th>Voltage</th>
                 <th>Temp</th>
             </tr>
-            <tr>
+            {% for i in range(20) %}
+            <tr id="cell-{{ i }}">
```

Comment:
I have no idea why I didn't do this

Reply:
Please take a moment to appreciate how stupid I was

Reply:
Noone thinks straight that close to finals